flask_common/mongo/utils.py

```diff
@@ -19,8 +20,13 @@ def iter_no_cache(query_set):
     if query_set._batch_size is None:
         query_set = query_set.batch_size(1000)
 
+    next = query_set.next if PY2 else query_set.__next__
```

Comment:
Can we use `from builtins import next` instead?

Reply:
We'll have to change it anyway when we move to upstream. And it's a bit [complicated](https://github.com/PythonCharmers/python-future/blob/master/src/future/builtins/newnext.py#L54) and potentially more resource-consuming (nested `try`) for something that's called in a tight loop.